VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+
+### Do validators need to self-bond Atoms?
+
+No, they do not. A validators total stake is equal to the sum of its own self-bonded stake and of its delegated stake. This means that a validator can compensate its low amount of self-bonded stake by attracting more delegators. This is why reputation is very important for validators. 
+
+Even though there is no obligation for validators to self-bond Atoms, delegators should want their validator to have self-bonded Atoms in their staking pool. In other words, validators should have skin in the game. 
+
+In order for delegators to have some guarantee about how much skin-in-the-game their validator has, the latter can signal a minimum amount of self-bonded Atoms. If a validator's self-bond goes below the limit that it predefined, this validator and all of its delegators will unbond. 
+
+### How to prevent concentration of stake in the hands of a few top validators?
+
+For now the community is expected to behave in a smart and self-preserving way. When a mining pool in Bitcoin gets too much mining power the community usually stops contributing to that pool. The Cosmos Hub will rely on the same effect initially. In the future, other mechanisms will be deployed to smoothen this process as much as possible:
+
+* **Penalty-free re-delegation:** This is to allow delegators to easily switch from one validator to another, in order to reduce validator stickiness.
+* **Hack bounty:** This is an incentive for the community to hack validators. There will be bounties proportionate to the size of the validator, so that a validator becomes a bigger target as its stake grows.
+* **UI warning:** Users will be warned by the Cosmos UI if they want to delegate to a validator that already has a significant amount of staking power.
+
+### Will validators of the Cosmos Hub ever be required to validate other zones in the Cosmos ecosystem?
+
+It is highly probable that they will. If governance decides so, validators of the Cosmos hub might have to validate other zones in the Cosmos ecosystem. For example, validators of the Cosmos hub will also validate the first public Ethermint zone. Of course, they will be compensated for it in the form of block provisions and transaction fees. These rewards will be paid in the native currency of the Ethermint zone, called `photons`.
+
+
+## Technical requirements 
+
+### How can I join the testnet?
+
+The Testnet is a great environment to test your validator setup before launch.
+
```

Comment:
We should promote this much sooner. Maybe at the top. It's too far down the document. We can have it here as well but the goal of our materials needs to be to funnel people to testnets

Reply:
Moved it up